frontend/local/src/modules/ehr/encounter.js

```diff
@@ -154,12 +184,24 @@ export default dispatch =>
             unit: "/min",
             formPath: "vitalSigns.heart_rate.value"
         },
+        {
+            type: "dateTime",
+            ehrPath:
+                "/content[openEHR-EHR-COMPOSITION.encounter.v1]/context/other_context/items[openEHR-EHR-OBSERVATION.pulse.v1]/context/start_time",
+            formPath: "vitalSigns.pulse.timestamp"
+        },
```

Comment:
events requires a the same `:0` suffix as the rest of observations. timestamp should be stored in the same way as the measurements above.